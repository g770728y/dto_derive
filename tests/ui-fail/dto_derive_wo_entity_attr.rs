extern crate dto_derive;
use dto_derive::Dto;

#[derive(Debug, PartialEq, Dto)] //~ ERROR required `entity` attribute
struct DtoRequest { }

#[derive(Debug, PartialEq, Dto)] //~ ERROR required `entity` attribute
struct DtoResponse { }

#[derive(Debug, PartialEq, Dto)] //~ ERROR required `entity` attribute
struct Dto { }

fn main() { }
