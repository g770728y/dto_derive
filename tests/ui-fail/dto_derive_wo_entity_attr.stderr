error: required `entity` attribute
 --> $DIR/dto_derive_wo_entity_attr.rs:4:28
  |
4 | #[derive(Debug, PartialEq, Dto)] //~ ERROR required `entity` attribute
  |                            ^^^

error: required `entity` attribute
 --> $DIR/dto_derive_wo_entity_attr.rs:7:28
  |
7 | #[derive(Debug, PartialEq, Dto)] //~ ERROR required `entity` attribute
  |                            ^^^

error: required `entity` attribute
  --> $DIR/dto_derive_wo_entity_attr.rs:10:28
   |
10 | #[derive(Debug, PartialEq, Dto)] //~ ERROR required `entity` attribute
   |                            ^^^

error: aborting due to 3 previous errors

